### PR TITLE
update to autoprefixer 6

### PR DESCRIPTION
**Pull request by neilhem**

### CODE REVIEW

File: app/templates/gulpfile.babel.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ gulp.task('styles', () => {<% if (includeSass) { %>
     }).on('error', $.sass.logError))<% } else { %>
   return gulp.src('app/styles/*.css')
     .pipe($.sourcemaps.init())<% } %>
-    .pipe($.autoprefixer({browsers: ['last 1 version']}))
+    .pipe($.autoprefixer({browsers: ['> 1%', 'last 2 versions', 'Firefox ESR']}))
```
Comment (silvenon) [inline app/templates/gulpfile.babel.js]: Why is this change a part of updating Autoprefixer?

Comment (neilhem) [inline app/templates/gulpfile.babel.js]: I have set default browsers as it listed in [browserlist](https://github.com/ai/browserslist#browserslist-)

Comment (silvenon) [inline app/templates/gulpfile.babel.js]: Right, it's best to keep them in sync.

**Comment by silvenon** [comment]
Thanks!

**Pull request merged**